cni-plugin/pkg/types/types.go

```diff
@@ -123,6 +123,9 @@ type NetConf struct {
 	// Options below here are deprecated.
 	EtcdAuthority string `json:"etcd_authority"`
 	Hostname      string `json:"hostname"`
+
+	//Cni waits until all the endpoints specified in ReadinessGates are ready
```

Comment:
Just a nit, but comments in the Calico codebase should have a space after the `//` and `Cni` should be `The CNI plugin` 

Also, you should add this above the section that says "Options below here are deprecated"

Reply:
Modified.